docs/source/tutorial/registration.md

```diff
@@ -74,7 +78,9 @@ The common loss function is Dice loss, Jacard and average cross-entropy over all
 
 ### Deformation loss
 
-This type of loss measures the deformation to regularize the transformation.
+This type of loss measures the amount of deformation and penalises severe deformations.
```

Comment:
I would also be more specific here: "penalise the non-smooth deformation", as in theory some deformation, e.g. dT/d(x,y,z)=0 will not be penalised.

Reply:
Changed to: "This type of loss measures the amount of deformation in an image and penalises
non-smooth deformations. Penalising sudden or discontinuous deformations helps to
regularise the transformation between fixed and moving images."